### PR TITLE
first-commit

### PR DESCRIPTION
**Pull request by daginwu**
go server test fine
[wallets] pass
[try] pass
[confirm] stuck
[cancel] not test

main use pkg 
[gofiber]
[twist]

### CODE REVIEW

**Comment by daginwu** [comment]
done!

**Comment by daginwu** [comment]
check again after dinner fix a taskcancel bug

**Comment by daginwu** [comment]
done both side

**Comment by daginwu** [comment]
add config file

**Comment by daginwu** [comment]
add config file

**Comment by daginwu** [comment]
weird bug QQ

File: server/app/api/deduct.go [hunk 1]
```diff
@@ -28,17 +27,25 @@ func InitDeductAPI(r *gin.Engine) {
 		}
 
 		if request.User == "" {
-			log.Fatal("Need user")
+			c.JSON(http.StatusOK, gin.H{
```
Comment (JheSue) [inline server/app/api/deduct.go]: return StatusBadRequest

Comment (daginwu) [reply server/app/api/deduct.go]: fix~

**Review by JheSue — APPROVED** [review]
Cool!

**Pull request merged**